content/en/docs/tasks/observability/mixer/metrics/tcp-metrics/index.md

```diff
@@ -17,8 +22,34 @@ as the example application throughout this task.
 
 ## Before you begin
 
-* [Install Istio](/docs/setup) in your cluster and deploy an
-application.
+* [Install Istio](/docs/setup) with Mixer enabled in your cluster and deploy an application.
+
+    The *custom* configuration needed to use Mixer for telemetry is:
+
+    {{< text yaml >}}
+    values:
+      prometheus:
+        enabled: true
+      telemetry:
+        v1:
+          enabled: true
+        v2:
+          enabled: false
+    components:
+      telemetry:
```

Comment:
I also had to enable `citadel` otherwise `istio-telemetry` would go into a crash look backoff while it was `waiting for file` before it would timeout and exit.

Reply:
you need citadel or disable control plane security

Reply:
John is correct. I tried the following to verify all combinations.

| Citadel installed | ControlPlaneSecurityEnabled | Works? |
|-------------------|-----------------------------|--------|
| Yes               | Yes                         | Yes    |
| Yes               | No                          | Yes    |
| No                | Yes                         | No     |
| No                | No                          | Yes    |

So it's only the default installation that doesn't work.

Reply:
@howardjohn @jacob-delgado There is not option that the istio-telemetry v1 will work with SDS support instead of disabling `ControlPlaneSecurityEnabled` or installing Citadel?

Reply:
in theory yes. it's not implemented

Reply:
@howardjohn @jacob-delgado I adopted those settings from https://github.com/istio/istio/blob/master/tests/integration/mixer/telemetry/metrics/scenarios_test.go#L198-L212. Is the assertion that those tests disable control plane security then?

Reply:
I guess this also means that `istio-citadel` is actually *not* in the profiles listed in `preliminary.istio.io/docs/setup/additional-setup/config-profiles/` ?

Reply:
OK. Updated. PTAL.